uwblueprint/sistering
# Remove unused profile photo upload button + text

## Ticket link

Closes #534


## Implementation description
* removes the remove profile picture button and the default crab profile picture in ProfilePhotoForm.tsx


## Steps to test
1. Navigate to http://localhost:3000/edit-account, verify the ProfilePhotoForm displays the following
<img width="1175" alt="Screen Shot 2022-08-13 at 3 29 39 PM" src="https://user-images.githubusercontent.com/82241706/184515441-579b8937-f980-4d05-9f4c-752974659952.png">


Note: Also need to confirm with Anthea whether the profile icon is valid 


## What should reviewers focus on?
* Verify that the changes correspond to what is required for the ticket 


## Checklist
